app/models/appointment_summary.rb

```diff
@@ -8,7 +8,13 @@ class AppointmentSummary < ActiveRecord::Base
             inclusion: { in: Partners.delivery_partners },
             uniqueness: { scope: :reporting_month }
   validates :reporting_month, format: REPORTING_MONTH_REGEXP
-  validates :completions, numericality: { less_than_or_equal_to: ->(as) { as.transactions } }
+  validates :completions,
+            numericality: {
+              greater_than_or_equal_or: 0,
```

Comment:
Same here. Also, do these two work together when specified for numericality? Can we not use a range instead?